### DISCUSSION TITLE
A world map for #MahsaAmini and IRAN

### DISCUSSION THREAD

**Discussion by Setareh4IRAN** [WeAreMahsaAmini/Core#158] [Ideas]
A map to attach and follow all relevant events, demonstrations around the world .
so that more people will be aware and attend and also maybe we can create better contents from these events. [like berlin or vancouver videos]

**Comment by sinahnyazdi** [comment]
They built one here: https://iranrally.ushahidi.io/views/map
and also ongoing discussions for integration here: #37
